### PR TITLE
For #7468 - Don't set notification style on Huawei devices on lollipop

### PR DESCRIPTION
**Pull request by ekager**
---

### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

File: components/feature/media/src/main/java/mozilla/components/feature/media/notification/MediaNotification.kt [hunk 1]
```diff
@@ -21,6 +22,7 @@ import mozilla.components.feature.media.ext.nonPrivateIcon
 import mozilla.components.feature.media.ext.nonPrivateUrl
 import mozilla.components.feature.media.service.AbstractMediaService
 import mozilla.components.support.base.ids.SharedIdsHelper
+import java.util.*
```
Comment (NotWoods) [inline components/feature/media/src/main/java/mozilla/components/feature/media/notification/MediaNotification.kt:25]: Looks like this got included by accident

Comment (ekager) [reply components/feature/media/src/main/java/mozilla/components/feature/media/notification/MediaNotification.kt]: the star import is an accident but it needs the locale package 😢

**Comment by ekager** [comment]
bors r=NotWoods

**Comment by ekager** [comment]
bors retry

**Pull request merged**